src/core/ext/filters/client_channel/resolver/dns/c_ares/grpc_ares_ev_driver_libuv.cc

```diff
@@ -176,4 +176,4 @@ UniquePtr<GrpcPolledFdFactory> NewGrpcPolledFdFactory(grpc_combiner* combiner) {
 
 }  // namespace grpc_core
 
-#endif /* GRPC_ARES == 1 && defined(GRPC_UV) */
\ No newline at end of file
+#endif /* GRPC_ARES == 1 && defined(GRPC_UV) */
```

Comment:
Did some other change break clang formatting everywhere? It seems odd that this PR would cause formatting across the build.